.github/workflows/ci.yml

```diff
@@ -345,6 +357,10 @@ jobs:
         if: matrix.rust == 'nightly-2024-02-13'
 
       - run: tools/test.sh -vv ${TARGET:-} ${DOCTEST_XCOMPILE:-} ${BUILD_STD:-} ${RELEASE:-}
+      - run: tools/test.sh -vv ${TARGET:-} ${BUILD_STD:-} ${RELEASE:-}
+        env:
+          RUSTFLAGS: ${{ env.RUSTFLAGS }} --cfg portable_atomic_coerce_unsized
```

Comment:
I think this should be named portable_atomic_unstable_coerce_unsized as with other unstable feature related cfgs.

https://github.com/taiki-e/portable-atomic/blob/16816f4b9f526bbae180472a818274e9e1734b51/src/lib.rs#L234-L243

Reply:
My bad, I will fix this thanks!

Reply:
Should be fixed now.